UI/obs-app.cpp

```diff
@@ -429,7 +429,7 @@ bool OBSApp::InitGlobalConfigDefaults()
 
 	if (!config_get_bool(globalConfig, "General", "Pre21Defaults")) {
 		config_set_default_string(globalConfig, "General",
-					  "CurrentTheme", DEFAULT_THEME);
+					  "CurrentTheme2", DEFAULT_THEME);
```

Comment:
This change shouldn't be necessary.  You can safely remove this.

Reply:
Er, didn't specify why:  this change will cause the theme to reset to dark when upgrading.  It won't preserve the old value between versions.

Reply:
New versions will still check that old key. https://github.com/obsproject/obs-studio/blob/ce17c1c9ec2e777f99e824dc5a7678a9d5cada2e/UI/obs-app.cpp#L1077-L1079

Reply:
That line is what's causing upgrades to break version.  That change definitely isn't necessary if you look at the way the fallback is handled.  It can basically use the default from that value safely.  If you step through the code you'll see what I mean.

Reply:
Change made.